components/automate-ui/src/app/modules/user/user-details/user-details.component.html

```diff
@@ -30,9 +30,27 @@
               Display Name is required.
             </chef-error>
           </chef-form-field>
+          <chef-form-field class="timezone-field">
+            <label for="timeformat">Time Format</label>
+            <chef-select 
+            #timeformat 
+            [value]="(userPrefsService.timeformat$ | async)?.value" 
+            (change)="handleTimeFormatChange()" 
+            data-cy="timeformat-dropdown">
+              <chef-option value="ddd, DD MMM">ddd, DD MMM</chef-option>
+              <chef-option value="ddd, DD MMM YYYY">ddd, DD MMM YYYY</chef-option>
+              <chef-option value="DD MMM YYYY">DD MMM YYYY</chef-option>
+              <chef-option value="ddd, DD MMM YYYY HH:mm:ss [UTC]">ddd, DD MMM YYYY HH:mm:ss [UTC]</chef-option>
+              <chef-option value="YYYY-M-D">YYYY-M-D</chef-option>
+              <chef-option value="YYYY-MM-DD">YYYY-MM-DD</chef-option>
+            </chef-select>
+          </chef-form-field>
         </form>
-        <chef-button [disabled]="(userDetails.loading$ | async) || !userDetails.displayNameForm.valid || !userDetails.displayNameForm.dirty" primary inline
-          (click)="userDetails.saveDisplayName()" data-cy="user-details-submit-button">
+        <chef-button 
```

Comment:
it should be aligned properly. 
follow this : https://github.com/chef/automate/blob/b8486c44deeb30362f8691fe1fa5f5025d60e042/components/automate-ui/src/app/modules/infra-proxy/infra-roles/infra-roles.component.html#L3-L10

Reply:
done